app/src/test/java/com/example/foody/recipe_search/presentation_mvvm/SearchViewModelTest.kt

```diff
@@ -0,0 +1,189 @@
+package com.example.foody.recipe_search.presentation_mvvm
+
+import app.cash.turbine.ReceiveTurbine
+import app.cash.turbine.turbineScope
+import com.example.foody.recipe_search.domain.RecipesSearchRepository
+import com.example.foody.recipe_search.presentation_mvvm.model.RecipeListState
+import com.example.foody.recipe_search.presentation_mvvm.model.SearchScreenState
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Assert.*
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.io.IOException
+
+class SearchViewModelTest {
+    @OptIn(ExperimentalCoroutinesApi::class)
+    private var dispatcher = UnconfinedTestDispatcher()
+    
+    private lateinit var subject: SearchViewModel
+    private lateinit var repoMock: RecipesSearchRepository
+    private lateinit var testSearchTerm: String
```

Comment:
This does not need to be a field as a lateinit var.
lateinit var we set within `setup` function are setup before each test. Since it is the same for each test, we do it once in setup, so we don' t need to do it every time in each test.
 Since this is specific for each test, it makes more sense to set it as a val within each test.

Reply:
Just the searchTerm?

Reply:
I was referring to the `testSearchTerm` only, yes.